.github/workflows/ci.yml

```diff
@@ -69,11 +73,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
+      with:
+        fetch-depth: 0
```

Comment:
Actually it seemed to work without this (very nice!!!), just silences a few warnings.